alembic/versions/2024_01_24_a6befebf1231_add_order_table.py

```diff
@@ -0,0 +1,48 @@
+"""Add order table
+
+Revision ID: a6befebf1231
+Revises: 584840c706a0
+Create Date: 2024-01-24 10:43:55.206551
+
+"""
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "a6befebf1231"
+down_revision = "584840c706a0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "order",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("customer_id", sa.Integer(), nullable=True),
+        sa.Column("order_date", sa.DateTime(), nullable=True),
+        sa.Column("ticket_id", sa.Integer(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["customer_id"],
+            ["customer.id"],
+        ),
+        sa.PrimaryKeyConstraint("id"),
+        sa.UniqueConstraint("id"),
```

Comment:
Would we want this to reference the customer internal_id since it otherwise won't show in statusdb?